.changeset/famous-trees-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-invoices": patch
+---
+
+Improved error handling in Webhook INVOICE_CREATED. Now Sentry will gather additional breadcrumbs for better debugging. No PII is logged
```

Comment:
```suggestion
Improved error handling in Webhook INVOICE_CREATED. Now Sentry will gather additional breadcrumbs for better debugging.
```

I think we can omit that, it's an implementation detail.

Reply:
No, its a privacy policy and security detail. Logs are sent to 3rd party service so its not implementation